bluebottle/bb_accounts/views.py

```diff
@@ -64,6 +65,22 @@ class ManageProfileDetail(generics.RetrieveUpdateAPIView):
     permission_classes = (TenantConditionalOpenClose, IsCurrentUser)
     serializer_class = ManageProfileSerializer
 
+    def pre_save(self, obj):
+        try:
+            # Read only properties come from the TOKEN_AUTH / SAML settings
+            assertion_mapping = properties.TOKEN_AUTH['assertion_mapping']
```

Comment:
Why doesn't this use the readonly fields settings from the previous pull request?